.buildkite/pipeline.yml

```diff
@@ -91,6 +99,22 @@ steps:
       distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
       hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
 
+  - label: ':hammer: test against firecracker master'
+    env:
+      FC_TEST_BIN: "testdata/firecracker-master"
+      FC_TEST_JAILER_BIN: "testdata/jailer-master"
+    commands:
+      - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
+      - 'ln -s /var/lib/fc-ci/rootfs.ext4 testdata/root-drive.img'
+      - 'make -C cni install CNI_BIN_ROOT=$(pwd)/testdata/bin'
+      - "sudo FC_TEST_TAP=fc-root-tap${BUILDKITE_BUILD_NUMBER} make test EXTRAGOARGS='-v -count=1 -race' DISABLE_ROOT_TESTS="
+    agents:
+      queue: "${BUILDKITE_AGENT_META_DATA_QUEUE:-default}"
+      distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
+      hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
+    soft_fail:
+      - exit_status: 2
```

Comment:
Why?

Reply:
We don't want failures to block PRs if master is broken. It should just let us know that something is broken in master.

Reply:
I see. Thanks.